src/packages.builds

```diff
@@ -24,9 +24,6 @@
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
     <!-- add specific builds / pkgproj's here to include in servicing builds -->
-    <Project Include="$(MSBuildThisFileDirectory)\System.Data.SqlClient\pkg\System.Data.SqlClient.pkgproj">
```

Comment:
You don't have to delete it. If you delete it, it won't be built, and we have to produce the stable package of this version. So only adding the version that we want to be stable into packageIndex.json and leave the pkgproj in this .builds file so that it is built.

Reply:
Gotcha, will fix